src/GeneratorV2/Process/Overloaders.cs

```diff
@@ -271,6 +278,9 @@ public bool TryGenerateOverloads(Overload overload, [NotNullWhen(true)] out List
                     }
                     else if (parameter.Length is BinaryOperation binary)
                     {
+                        // We need to figure out which operator side is the length of the vector type.
+                        // and which side is the parameter reference.
+                        // The side with the parameter reference, references the length parameter of an array.
```

Comment:
This comment kind of assumes you have the contextual information that the gl.xml spec contains length entries where there is a parameters name multiplied by a vector length, but I think it's mostly fine.